File: BackEndApplication/KanBanBoardApplication/EmployeeTask/src/main/java/com/EmployeeTask/EmployeeTask/domain/Task.java
package com.EmployeeTask.EmployeeTask.domain;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.util.Set;

@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class Task {
    private int taskID;
    private String taskName;
    private String taskDescription;
    private String taskCreatedDate;
    private String taskEndDate;
    private String taskPriority;
    private String taskStage;
    private String taskCreatedBY;
    private Set<String> assignTaskEmployees;
}

File: Applications/Application/src/app/Model/task.ts
export interface ITask {
    taskID: number;
    taskName: string;
    taskDescription: string;
    taskCreatedDate: string;
    taskEndDate: string;
    taskPriority: string;
    taskStage: string;
    taskCreatedBY: string;
    assignTaskEmployees: string[];
}
